tests/unit/test_charm_2.py

```diff
@@ -126,14 +173,26 @@ def mock_test_database_connectivity(_self):
             except mysql.connector.Error as err:
                 return False, f"MySQL error {err.errno}"
 
+        def mock_apache_config_is_enabled(_self, conf_name: str) -> bool:
+            return conf_name in self._enabled_apache_conf
+
+        def mock_apache_enable_config(_self, conf_name: str, conf) -> None:
+            self._enabled_apache_conf.add(conf_name)
+
+        def mock_apache_disable_config(_self, conf_name: str) -> None:
+            self._enabled_apache_conf.remove(conf_name)
+
         self._patch = unittest.mock.patch.multiple(
             WordpressCharm,
             _current_wp_config=mock_current_wp_config,
             _push_wp_config=mock_push_wp_config,
             _remove_wp_config=mock_remove_wp_config,
             _run_wp_cli=mock_run_wp_cli,
             _test_database_connectivity=mock_test_database_connectivity,
-            _DB_CHECK_INTERVAL=0
+            _DB_CHECK_INTERVAL=0,
+            _apache_config_is_enabled=mock_apache_config_is_enabled,
+            _apache_enable_config=mock_apache_enable_config,
+            _apache_disable_config=mock_apache_disable_config
```

Comment:
Can you add a trailing "," here so future additions won't require a line change here?

Reply:
Fixed, thanks!